extensions/amp-a4a/0.1/callout-vendors.js

```diff
@@ -0,0 +1,11 @@
+//////////////////////////////////////////////////////////////////
+//                                                              //
+//     IMPORTANT: All keys in RTC_VENDORS must be lowercase     //
+//       otherwise the vendor endpoint will not be used.        //
+//                                                              //
+//////////////////////////////////////////////////////////////////
+/** @const {!Object<string, string>} */
+export const RTC_VENDORS = {
+  //'fakevendor': 'https://localhost:4443/posts?slot_id=SLOT_ID&page_id=PAGE_ID',
```

Comment:
is this line to be removed before submitting?

Reply:
removed